infra/main.bicep

```diff
@@ -123,6 +126,9 @@ module apim 'core/gateway/apim.bicep' = {
 module api 'app/apim-api.bicep' = {
   name: 'api'
   scope: resourceGroup
+  dependsOn: [
```

Comment:
not needed, as there are already params depending on both these resources. Bicep linter should report it